alert-common/src/main/java/com/synopsys/integration/alert/common/AlertProperties.java

```diff
@@ -136,8 +137,8 @@ public Boolean getH2ConsoleEnabled() {
         return h2ConsoleEnabled;
     }
 
-    public Boolean getSslEnabled() {
-        return sslEnabled;
+    public boolean getSslEnabled() {
+        return BooleanUtils.isTrue(sslEnabled);
```

Comment:
SonarCloud pushes to use boolean when possible to avoid issues with `null` objects